cmd/module/cmd.go

```diff
@@ -8,24 +8,20 @@ import (
 	"go.viam.com/rdk/module"
 
 	"github.com/erh/viamrtsp"
+	"go.viam.com/utils"
 )
 
 func main() {
-	err := realMain()
-	if err != nil {
-		panic(err)
-	}
+	utils.ContextualMain(mainWithArgs, module.NewLoggerFromArgs("client"))
```

Comment:
I changed this as the current implementation doesn't call `Close` on itself when the process receives certain signals which resulted in the RTSP server the module was connected to having dangling connections after the module died or rebooted. 

[ContextualMain](https://github.com/viamrobotics/goutils/blob/main/runtime.go#L39) will catch these signals & will close the context & allow the module to call `Close()` on itself & all models (which can then release RTSP connections).